plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -330,9 +330,15 @@ private static String quotedName(String name)
         return '"' + name.replace("\"", "\"\"") + '"';
     }
 
-    public static boolean canEnforceColumnConstraintInAllSpecs(TypeOperators typeOperators, Table table, IcebergColumnHandle columnHandle, Domain domain)
+    public static boolean canEnforceColumnConstraintInAllSpecs(
```

Comment:
I know it's not directly related, but shouldn't this function be marked as `@VisibleForTesting`? I was wondering why it was public

Reply:
This (and all the helpers below) are used by IcebergMetadata, so not VisibleForTesting exactly

Reply:
Ah... missed the file separator in the PR